panel/src/components/Forms/Upload.vue

```diff
@@ -95,6 +95,8 @@ export default {
     },
     upload(files) {
       this.$refs.dialog.open();
+      this.$store.dispatch("isLoading", true);
+      this.$store.dispatch("notification/success", "Uploading...");
```

Comment:
Using `upload.progress` translation instead of static text?

Reply:
Of course! Good catch